draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1068,6 +1108,44 @@ Disclosures:
 
 {{examples/w3c-vc/disclosures.md}}
 
+## Example 4b - W3C Verifiable Credentials Data Model v1.1, using JSON-LD
+
+This example illustrates how the artifacts defined in this specification can be
```

Comment:
"How the artifacts defined in this presentation can be used to represent a Verifiable Credential according to W3C Verifiable Credentials Data Model v2.0 ..."

Reply:
paraphrased